### PR TITLE
Update meeting templates

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: -meeting-template.md [hunk 1]
```diff
@@ -1,57 +1,54 @@
...
-## Follow Up
+### Working Group Updates
+
+- [Business Planning][biz-wg]
+- [Finance][fin-wg]
+- [Governance][gov-wg]
+- [Infra & Ops][ops-wg]
 
-### Actionable Items
+### Project Updates
```
Comment (dcwalk) [inline -meeting-template.md]: Where do actionable items go?

Comment (benhylau) [reply -meeting-template.md]: Project board task tracking

**Comment by dcwalk** [comment]
It seems like this new template wasn't a straightforward "win" for all members. Can we sit on this until more folks are around?

File: -meeting-template.md (hunk 1 above)
Comment (patcon) [reply -meeting-template.md]: would be nice if there were a space, as sometimes project board might not be convenient. cultivator can deal with straggling todos in notes before closing out, but having one space for that adds clarity

Would suggest template have:

```
### Action Items
- [ ] cultivator: migrate action items to task board
- [ ] ...
```

Comment (patcon) [reply -meeting-template.md]: also, proposing --

1. person speaking:

> `patcon: this is a thing patcon said`

2. assignment of todo:

> - [ ] this is a thing for patcon to do (patcon)

We use (1) for everything now, and it's been ambiguous whether something is assigned or suggested by someone. That distinction could be made clear in template by demonstration, if seems sensible

File: -meeting-template.md [hunk 2]
```diff
@@ -1,57 +1,54 @@
...
-[:date: Calendar][calendar] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive] | [:notebook: Meeting Index][meetings]
-
-Time: ...  
-Location: https://meet.jit.si/hyphacoop  
-Attending: ...
-
-Notetaker: ...
-
-:raising_hand: Next: ... COPY FROM PREVIOUS WEEK  
-<sup>Bump to end on taking turn :)</sup>
```
Comment (patcon) [inline -meeting-template.md]: why remote this?

File: -meeting-template.md [hunk 3]
```diff
@@ -1,57 +1,54 @@
...
+- [Business Planning][biz-wg]
+- [Finance][fin-wg]
+- [Governance][gov-wg]
+- [Infra & Ops][ops-wg]
 
-### Actionable Items
+### Project Updates
 
-- [ ] ...
+- ...
```
Comment (patcon) [inline -meeting-template.md]:
```suggestion
- someone: ...
```

As suggested in other comment

**Comment by patcon** [comment]
Agree with dc. Happy to see it merged for now, but might propose we revert later if this board-focussed format is too rigid and disjointed

File: -meeting-template.md (hunk 2 above)
Comment (benhylau) [reply -meeting-template.md]: We are all familiar with the turn by now and want to condense the lines before actual notes. The many lines at the top feel disorienting.

**Comment by benhylau** [comment]
> Agree with dc. Happy to see it merged for now, but might propose we revert later if this board-focussed format is too rigid and disjointed (jumping around doc to make todos mid-call feels 👎 and jumping between board and doc feels doubly so)

What if we just do `#todo` for all actions, so no scrolling to bottom list, and cultivator goes thru doc in the end and fills kanban? That's what I did last time, and some related `#todos` became different points in same ticket. So I think that extra step seems useful.

**Comment by patcon** [comment]
That works! Did like the idea that "Actionable items" section could have some standard boilerplate todos for cultivator baked into it. Totally your call though.

Note: we'll lose the copy-paste of "actionable items" into the new agenda's "previous action items", which made review quick. Is that a loss? I suppose that resetting the "done" column each meeting could allow for the same sort of review (since all "done" tickets will be since last time)

**Comment by benhylau** [comment]
It's the copy pasting of action items that I want to move away from. It doesn't allow for discussions to happen on those points in between meetings. I rather make better use of task board and when we all look at a task, we have all information relating to that task there. Even this split between Loomio and task tracker is a little sad, I feel it would be nice if Loomio has a portable module that can render directly into a GitHub issue it'd be perfect. Like disqus.

**Comment by patcon** [comment]
Makes sense.

And samesies on loomio-github split being confusing. Hoping we figure out better divides or "framing questions to ask ourselves", that make it simpler to know what should go where

(Far future solution could involve loomio api and probot, if we don't find simpler way that satisfies us.)

File: -standup-template.md [hunk 4]
```diff
@@ -1,19 +1,11 @@
...
-Notetaker: ...
-
-:raising_hand: Next: ... COPY FROM PREVIOUS WEEK  
-<sup>Bump to end on taking turn :)</sup>
-
-[TOC]
+Time:      ...  
+Location:  https://meet.jit.si/hyphacoop  
+Attending: ...  
+Notetaker: :raising_hand: ei, yj, gi, pc, dc, uv, bl
```
Comment (benhylau) [inline -standup-template.md]:
```suggestion
Notetaker: :raising_hand: el, yj, gi, pc, dc, uv, bl  
```

File: -standup-template.md [hunk 5]
```diff
@@ -22,17 +14,17 @@ Notetaker: ...
...
 
-- Standup
-  - @person
-    - last: ...
-    - next: ...
-    - blockers: ...
+- Person: ...
+	- last: 
+	- next: 
+	- blockers:
```
Comment (benhylau) [inline -standup-template.md]:
```suggestion

## Discussions
```

File: -meeting-template.md [hunk 6]
```diff
@@ -1,57 +1,54 @@
...
-Notetaker: ...
-
-:raising_hand: Next: ... COPY FROM PREVIOUS WEEK  
-<sup>Bump to end on taking turn :)</sup>
-
-[TOC]
+Time:      ...  
+Location:  https://meet.jit.si/hyphacoop  
+Attending: ...  
+Notetaker: :raising_hand: ei, yj, gi, pc, dc, uv, bl
```
Comment (benhylau) [inline -meeting-template.md]:
```suggestion
Notetaker: :raising_hand: el, yj, gi, pc, dc, uv, bl  
```

File: -meeting-template.md [hunk 7]
```diff
@@ -1,57 +1,54 @@
...
-    - [Infra & Ops][ops-wg]
-- Discussion
-    ...
-
-## Announcements
-
-- ...
+| Item                                            | Time (min) |
+|:------------------------------------------------|-----------:|
+| [Previous Action Items][tasks]                  |          5 |
```
Comment (benhylau) [inline -meeting-template.md]:
```suggestion
| [Previous Action Items][tasks]                  |         30 |
```

File: -meeting-template.md [hunk 8]
```diff
@@ -1,57 +1,54 @@
...
+| [Announcements](#Announcements)                 |          5 |
+| [Working Group Updates](#Working-Group-Updates) |         10 |
+| [Project Updates](#Project-Updates)             |         10 |
+| [Discussions](#Discussions)                     |         30 |
```
Comment (benhylau) [inline -meeting-template.md]:
```suggestion
| [Discussions](#Discussions)                     |         20 |
```

File: -meeting-template.md [hunk 9]
```diff
@@ -1,57 +1,54 @@
...
-- [ ] ...
+- ...
 
-### Upcoming Discussions
+### Discussions
 
 - ...
 
-<!-- Links -->
+
```
Comment (benhylau) [inline -meeting-template.md]:
```suggestion

### Action Items for Cultivator

- [ ] Migrate items with `#todo:person` to [task board][tasks]
- [ ] Publish meeting notes to [GitHub][gh]
- [ ] Publish [meeting template][template] for next [meeting][meetings]

```

**Comment by benhylau** [comment]
Can we merge this for now and iterate from here? @dcwalk @patcon

**Pull request merged**